### ISSUE TITLE
Support authenticated end to end tests

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#195]
## What's the problem?

Our current configuration for Playwright tests does not support Firebase logins, meaning we can't test anything that requires login.

## What's the idea?

Change the configuration to use the Firebase emulator and then then use Playwright's authentication support to authenticate the user.

## Who benefits?

Every user who logs in, which is hopefully most of them! And certainly folks writing automated tests.

## Design ideas

There are some issues with IndexedDB support in Playwright that might make this hard:

https://github.com/microsoft/playwright/issues/11164

**Comment by annieli555** [comment]
Hi there @amyjko! I'd be interested in tackling this issue and adding a few authenticated tests (for example, the one I would have added for [Update project references when character name changes #817](https://github.com/wordplaydev/wordplay/pull/817)). It looks like the linked issue above was resolved earlier this year / Playwright now supports IndexedDB!

**Comment by amyjko** [comment]
That would be great! I'd love to expand our scope of end to end tests, and this is a big barrier. Assigning you.

**Comment by amyjko** [comment]
@annieli555 Are you still working on this? Some of this is built now.
